update the wording for net amortized cost in AWS (#24074)
DataDog/documentation: content/en/cloud_cost_management/aws.md

In contrast, other metrics show private discounts as separate, negative-valued line items with no resource attribution tags. Rather than attributing the discounts directly to usage, those metrics subtract discounts from the total cost.

> **Net amortized** costs provide the most accurate representation for cost allocation, with all savings applied directly to usage. Net cost metrics are available if your AWS account has privately negotiated enterprise discounts. If your account lacks negotiated enterprise discounts, the **Net Amortized** Cost will be equivalent to the **Amortized** Cost.

Reviewer: ```suggestion
**Net amortized** costs provide the most accurate representation for cost allocation, with all savings applied directly to usage. Net cost metrics are available if your AWS account has privately negotiated enterprise discounts. If your account does not have enterprise discounts, then **net amortized** cost and **amortized** cost are equivalent.
```
